src/mono/browser/runtime/hybrid-globalization/module-exports.ts

```diff
@@ -1,6 +1,21 @@
-export * from "./calendar";
-export * from "./culture-info";
-export * from "./helpers";
-export * from "./change-case";
-export * from "./locales";
-export * from "./collations";
+import { RuntimeHelpers } from "../types/internal";
+import { mono_wasm_get_calendar_info } from "./calendar";
+import { mono_wasm_change_case, mono_wasm_change_case_invariant } from "./change-case";
+import { mono_wasm_compare_string, mono_wasm_starts_with, mono_wasm_ends_with, mono_wasm_index_of } from "./collations";
+import { mono_wasm_get_culture_info } from "./culture-info";
+import { mono_wasm_get_locale_info, mono_wasm_get_first_day_of_week, mono_wasm_get_first_week_of_year } from "./locales";
+
+export function initHybrid (rh: RuntimeHelpers) {
```

Comment:
When do we call it? Rollup is too early - it does not have `runtimeHelpers` initialized. In `globals.ts` in `setRuntimeGlobals`?